dev_guide/builds.adoc

```diff
@@ -105,6 +105,36 @@ The following options are specific to the
 link:../architecture/core_concepts/builds_and_image_streams.html#source-build[S2I
 build strategy].
 
+[[force-pull]]
+
+=== Force Pull
+
+By default, if the builder image specified in the build configuration is available in the OpenShift internal registry, that image will be used.  However, to override the local image and refresh it from
```

Comment:
again it's not the openshift internal registry that's being checked initially, it's the local system.